From: Alexey Yakovenko <wakeroid@gmail.com>
Description: English grammar fix ‘allows to set’ → ‘allows setting’
Origin: upstream, https://github.com/Alexey-Yakovenko/deadbeef/commit/72dbe320e3f525a16f360f1b08d70d3f089b881d
Bug: https://code.google.com/p/ddb/issues/detail?id=1257
--- a/plugins/gtkui/deadbeef.glade
+++ b/plugins/gtkui/deadbeef.glade
@@ -8808,7 +8808,7 @@ Descending</property>
 	      <property name="left_margin">8</property>
 	      <property name="right_margin">8</property>
 	      <property name="indent">0</property>
-	      <property name="text" translatable="yes">This dialog allows to set custom title for any track. This is most useful for radio stations. An option to set the custom title is also present in the &quot;Add Location&quot; dialog. The title itself is visible in columns displaying the &quot;Artist&quot; metadata field. It should look like &quot;[custom] artist&quot; if the Artist field is present, or just &quot;custom&quot; otherwise.</property>
+	      <property name="text" translatable="yes">This dialog allows setting custom title for any track. This is most useful for radio stations. An option to set the custom title is also present in the &quot;Add Location&quot; dialog. The title itself is visible in columns displaying the &quot;Artist&quot; metadata field. It should look like &quot;[custom] artist&quot; if the Artist field is present, or just &quot;custom&quot; otherwise.</property>
 	    </widget>
 	    <packing>
 	      <property name="padding">0</property>
